src/client_test.ts

```diff
@@ -0,0 +1,8 @@
+import {DockerFileParser} from "./dockerfile_parser/parser";
```

Comment:
Just adding a comment here, as explained by @akashshinde this file has test preparation part.